src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerNameParserTest.php

```diff
@@ -117,6 +116,7 @@ public function testInvalidBundleName($bundleName, $suggestedBundleName)
 
         try {
             $parser->parse($bundleName);
+            $this->fail();
```

Comment:
I'd pass a message to the `fail()` method, something like "Expected an exception to be thrown."